src/components/ImgEditor/ImgEditor.tsx

```diff
@@ -4,11 +4,14 @@ import { useDropzone } from "react-dropzone";
 import { FieldValues, useFormContext } from "react-hook-form";
 import { ImgLink, Props } from "./types";
 import Icon from "components/Icon";
+import { humanize } from "helpers";
 import useImgEditor from "./useImgEditor";
 
 type Key = keyof ImgLink;
 const fileKey: Key = "file";
 
+const BYTES_IN_MB = 1e6;
```

Comment:
since this conversion is unlikely to change in our lifetimes, no need to have it in shared constants.